Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -105,6 +105,19 @@ public async ValueTask<TItem> UpdateAsync(TItem value, CancellationToken cancell
             return Items[value.Id] = value;
         }
 
+        /// <inheritdoc/>
+        public async ValueTask<IEnumerable<TItem>> UpdateAsync(IEnumerable<TItem> values, CancellationToken cancellationToken = default)
```

Comment:
Looks good, there are some tests for this class that would need updating if you don't mind.

Reply:
Thanks for the quick review. Will add tests and create a PR.

Reply:
Updated PR with tests.